### DISCUSSION TITLE
Show indexed PointField values of PointCloud2 messages in the 3D panel's Selected Object panel

### DISCUSSION THREAD

**Discussion by DavidXenakis** [foxglove/community#325] [Q&A]
When inspecting individual points for a PointCloud2 object in the 3D panel, it would be useful to see the indexed PointField values. Right now, the panel shows x, y, and z values, but it would be useful to see all of them. Currently, the other PointField values show up as an array, which is not useful. In the example below, I'd expect to see the "i" value associated with the individual point.

![Example](https://user-images.githubusercontent.com/3076080/157081739-59b1eee1-f56c-44d6-a55c-77bb6e80a0c6.png)

**Comment by jtbandes** [comment]
FWIW, webviz does have this feature:

<img width="286" alt="image" src="https://user-images.githubusercontent.com/14237/157102703-742792f2-fbd1-4963-a155-360fb01eea25.png">
